searx/engines/translated.py

```diff
@@ -58,7 +63,7 @@ def response(resp):
         'title': '[{0}-{1}] {2}'.format(
             resp.search_params['from_lang'][1],
             resp.search_params['to_lang'][1],
-            resp.search_params['query']),
+            resp.search_params['query'])
```

Comment:
Without the coma, there is a syntax error:
https://travis-ci.org/github/asciimoo/searx/jobs/721829981#L938